worlds/paint/__init__.py

```diff
@@ -0,0 +1,81 @@
+from typing import Dict, Any
+
+from BaseClasses import Tutorial, Region, Entrance
+from worlds.AutoWorld import World, WebWorld
+from .items import item_table, PaintItem, item_data_table
+from .locations import location_table, PaintLocation, location_data_table
+from .options import PaintOptions
+
+
+class PaintWebWorld(WebWorld):
+    theme = "partyTime"
+
+    setup_en = Tutorial(
+        tutorial_name="Start Guide",
+        description="A guide to playing Paint in Archipelago.",
+        language="English",
+        file_name="guide_en.md",
+        link="guide/en",
+        authors=["MarioManTAW"]
+    )
+
+    tutorials = [setup_en]
+
+
+class PaintWorld(World):
+    game = "Paint"
+    options_dataclass = PaintOptions
+    options = PaintOptions
+    location_name_to_id = location_table
+    item_name_to_id = item_table
+
+    def create_item(self, name: str) -> PaintItem:
+        item = PaintItem(name, item_data_table[name].type, item_data_table[name].code, self.player)
+        return item
+
+    def create_items(self):
+        starting_tools = ["Brush", "Pencil", "Eraser/Color Eraser", "Airbrush", "Line", "Rectangle", "Ellipse",
+                          "Rounded Rectangle"]
+        self.multiworld.push_precollected(self.create_item("Magnifier"))
+        self.multiworld.push_precollected(self.create_item(starting_tools[self.options.starting_tool]))
+        starting_tools.pop(self.options.starting_tool)
+        items_to_create = ["Free-Form Select", "Select", "Fill With Color", "Pick Color", "Text", "Curve", "Polygon"]
+        items_to_create += starting_tools
+        items_to_create += ["Progressive Canvas Width"] * 4
+        items_to_create += ["Progressive Canvas Height"] * 3
+        items_to_create += ["Progressive Color Depth (Red)"] * 7
+        items_to_create += ["Progressive Color Depth (Green)"] * 7
+        items_to_create += ["Progressive Color Depth (Blue)"] * 7
+        pre_filled = len(items_to_create)
+        to_fill = len(self.multiworld.get_region("Canvas", self.player).locations)
```

Comment:
```suggestion
        to_fill = len(self.get_region("Canvas").locations)
```
world.get_region is newer, would recommend using it over the multiworld version